api/grpc/mpi/v1/command.proto

```diff
@@ -41,8 +41,55 @@ service CommandService {
 message CreateConnectionRequest {
     // Meta-information associated with a message
     mpi.v1.MessageMeta message_meta = 1;
-    // instance information associated with the NGINX Agent
-    Instance agent = 2;
+    // instance and infrastructure information associated with the NGINX Agent
+    Resource resource = 2;
```

Comment:
Even though the type is `Resource`, maybe we still name the field `agent` ?

Reply:
it represents where the Agent is deployed and the associated information around that. Happy to adjust the name but just want to say that it includes Agent, any NGINX product(s), host or container information